test/syscall_test/blocklists.exfat/xattr_test

```diff
@@ -0,0 +1,34 @@
+XattrTest.XattrLargeName
```

Comment:
How many xattr tests are actually enabled? The block list seems to be quite long.

Reply:
This is the blocklist for our exfat, which doesn't support xattr yet. As for ramfs and ext2, 40 out of 41 tests passed, only 1 blocked.

Reply:
But the `blocklists/xattr_test` file, which is for Ext2 test, is also quite long.